src/decorator.ts

```diff
@@ -110,5 +123,10 @@ export default function ReactN<
   }
   */
 
+  if (isPureComponent(DecoratedReactNComponent.displayName)) {
+    // Must remove shouldComponentUpdate from prototype for pure components or it will get warning and side effects
```

Comment:
Is this still needed, since we aren't adding `shouldComponentUpdate` to decorated components? You commented out the method above. At this point, we are deleting a method added by a third party (React itself or the developer themselves) and not a method we created.